core/Cargo.toml

```diff
@@ -37,6 +37,12 @@ parallel = [
     "bls12_381-bls/parallel", 
 ]
 
+# These are the features available for when one wishes to use `rusk-abi` as a
```

Comment:
```suggestion
# These are the features available for when one wishes to use `rusk-abi` in a
```

Reply:
Actually we also need to scratch the `rusk-abi` part there.